app/admin/views.py

```diff
@@ -83,6 +83,79 @@ def approved_users():
         'admin/approved_users.html', users=users, roles=roles)
 
 
+@admin.route('/add-tag')
+@login_required
+@admin_required
+def add_tags():
+    """View and manage all tags."""
+    Tags = Tag.query.all()
+    form = AddTagForm()
+    age_group = TagType.query.filter_by(tag_type_name="Age Group").first()
+    service = TagType.query.filter_by(tag_type_name="Service").first()
+    disability_programming = TagType.query.filter_by(tag_type_name="Disability Programming").first()
+    return render_template(
+        'admin/add_tags.html',tags=Tags, form=form,age_group=age_group,service=service,
+        disability_programming=disability_programming)
+
+
+@admin.route('/add-tag', methods=['GET', 'POST'])
```

Comment:
not sure if this is a mess up in the merging process, but could you clarify which of these two is the correct meethod and remove the incorrect one?

Reply:
nevermind -- fixed

Reply:
yea so add_tag is just for viewing the add tag page, while the add_new_tag method is for when someone fills out the form. Lmk if that makes sense!

Reply:
The problem is that they have the same route name -- so I get an error when  I try to run it on my machine. I know it works completely on your machine so it may be an issue with merging branches. Either way, think there should be a "view-tags" route (and method) and then a separate "add-new-tag" route (with that method).